### PR TITLE
References, add active link

### LINKED ISSUE #11: Add <a href> to References links
Update template (example):

```
<li><cite id="bib-ietf-rfc-7405">IETF RFC 7405</cite>, “Case-Sensitive String Support in ABNF”, P. Kyzivat, December 2014.
<a href="https://datatracker.ietf.org/doc/html/rfc7405.txt">https://datatracker.ietf.org/doc/html/rfc7405.txt</a></li>
```

### PR DESCRIPTION
**Pull request by SteveLLamb**
Add href to references, closes #11

### CODE REVIEW

File: example/index.html [hunk 1]
```diff
@@ -464,7 +464,7 @@ <h2>Copyright and License Element</h2>
     draft-bhutton-json-schema-00</cite>, Wright, A., H. Andrews; B. Hutton, G.
     Dennis, "JSON Schema: A Media Type for Describing JSON Documents", Work in
     Progress, December 8, 2020,
-    https://datatracker.ietf.org/doc/html/draft-bhutton-json-schema-00
+    <a href="https://datatracker.ietf.org/doc/html/draft-bhutton-json-schema-00">https://datatracker.ietf.org/doc/html/draft-bhutton-json-schema-00</a>
```
Comment (palemieux) [inline example/index.html:467]: ?

File: example/index.html [hunk 2]
```diff
@@ -30,16 +30,16 @@ <h2>Scope</h2>
     <ul>
       <li><cite id="bib-ietf-rfc-3986">IETF RFC 3986</cite>, Uniform Resource
       Identifiers (URI): Generic Syntax, T. Berners-Lee, et al., January 2005.
-      http://www.ietf.org/rfc/rfc3986.txt</li>
+      <a href="http://www.ietf.org/rfc/rfc3986.txt">http://www.ietf.org/rfc/rfc3986.txt</a></li>
```
Comment (palemieux) [inline example/index.html:33]:
```suggestion
      url: <a href="http://www.ietf.org/rfc/rfc3986.txt">http://www.ietf.org/rfc/rfc3986.txt</a></li>
```

**Review by palemieux — CHANGES_REQUESTED** [review]

File: example/index.html (hunk 2 above)
Comment (SteveLLamb) [reply example/index.html]: addressed in https://github.com/SMPTE/html-pub/pull/12/commits/18c2b626d8a92c1fbfc7fd5faf76495e8cde912e, had to also add a `<br>` to force the URL to new line at end

File: example/index.html (hunk 1 above)
Comment (SteveLLamb) [reply example/index.html]: Not sure what the question is here, added the href to bib refs as well.

**Comment by palemieux** [comment]
@SteveLLamb Please review the proposed changes:

* it looks like the _url:_ label is no longer popular
* the tooling now auto-generates a `href` if a URI is wrapped in an `<a>` element
* URLs no longer wrap, so no need for `<br>`

**Comment by SteveLLamb** [comment]
> 

@palemieux I'm happy with these updates.

**Pull request merged**